src/servlets/metaTags/index.ts

```diff
@@ -98,7 +99,7 @@ const getTrackContext = async (
       tags,
       labels,
       image: track.artwork['1000x1000'],
-      embed: canEmbed,
+      embed: canEmbed && isPremium,
```

Comment:
Do we want this to be false if the track is premium?
```suggestion
      embed: canEmbed && !isPremium,
```